ball.java

```diff
@@ -0,0 +1,12 @@
+public class ball {
+    int number;
+    Colors color;
+    public ball(int number, Colors color){
+        this.number = number;
+        this.color = color;
+    }
+
+    public String toString(){

```

Comment:
The `toString` method should return a more descriptive string. Consider formatting it as `"Ball{color='" + color + "', number=" + number + "}"` to clearly indicate the properties of the ball.